pypeit/spectrographs/keck_hires.py

```diff
@@ -298,7 +301,7 @@ def compound_meta(self, headarr, meta_key):
                 if headarr[0].get('HATOPEN') and headarr[0].get('AUTOSHUT'):
                     return 'Object'
                 elif not headarr[0].get('HATOPEN'):
-                    return 'Bias' if not headarr[0].get('AUTOSHUT') else 'Dark'
+                    return 'Bias' if headarr[0].get('ELAPTIME') == 0 else 'Dark'
```

Comment:
Just checking the use of equality here.  Given your note above, is this equivalent to, say, `headarr[0].get('ELAPTIME') < 0.1`.  Just want to make sure we don't get caught by numerical precision errors.

Reply:
Only thought I had (why I left it in draft) was that this now no longer follows the exprng set in the spectrograph configuration, although I didn't see an elegant way to preserve that behavior. Are we OK with that outcome?

Reply:
Yeah, I think that's fine.